vendordeps/Phoenix.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
This extra `vendordeps` directory should be removed. The one inside the inner `2020-InfiniteRecharge` folder is the right one.